activesupport/lib/active_support/i18n_railtie.rb

```diff
@@ -4,6 +4,22 @@
 
 module I18n
   class Railtie < Rails::Railtie
+    module Helpers
+      def self.extensions_for(glob)
+        $1.split(',') if glob =~ /\{([\S]+)\}/
```

Comment:
I still don't understand why this is needed. As @rafaelfranca said in https://github.com/rails/rails/pull/21124/files#r36254668:

>> We can handle new files inside app without having to change `Paths`, so I believe we can use the same machinery to load new locales files.

@kirs can you elaborate on why we can't use the existing glob'ing in `Paths` and why this extensions thing is needed? :smile: